src/core/server/models/story/index.ts

```diff
@@ -120,14 +123,14 @@ export interface UpsertStoryInput {
 export async function upsertStory(
   mongo: Db,
   tenantID: string,
-  { id, url }: UpsertStoryInput,
+  { id = uuid.v4(), url }: UpsertStoryInput,
```

Comment:
I've noticed we're using `uuid.v1()` in some places and `uuid.v4()` in others. Would it make sense to pick one and standardize across the codebase?

Reply:
`uuid.v1()` is used when a true "secure random" source isn't required, this is for things like log entries and such. This should be preferred because of the lower cost on the underlying machine versus `uuid.v4()`, which has a higher cost to call.

Reply:
I was aware of differing cost to generate for v4 versus v1, wasn't aware it was of a large enough impact we were worried about it. I would have used v4 everywhere and not worried about it until it became a bottleneck.

Good to know, will keep in mind for future.